source/core/Timestamp.cpp

```diff
@@ -163,6 +163,14 @@ namespace Vireo
         return answer;
     }
     //------------------------------------------------------------
+    Timestamp const Timestamp::operator-(const Int64& value)
```

Comment:
This is ok, but it is a good idea to invoke operator+() directly with "-value" as an argument.